demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -120,6 +120,25 @@ def is_docker_image_same_as_yml(self) -> bool:
 
         return True
 
+    def is_breaking_change(self) -> bool:
+        """
+        In case of a breaking change in the release note, ensure the existence of a proper json file.
+        """
+        is_valid = True
+        if 'breaking change' in self.latest_release_notes.lower():
+            json_path = self.release_notes_file_path[:-2] + 'json'
+            error_message, error_code = Errors.release_notes_bc_json_file_missing(json_path)
+            try:
+                js = get_dict_from_file(path=json_path)[0]  # extract only the dictionary
+                error_message, error_code = Errors.release_notes_bc_json_file_missing(json_path)
```

Comment:
```suggestion
```

Reply:
Can you please explain the reason? In the [last CR](https://github.com/demisto/demisto-sdk/pull/2046#discussion_r869210826) you told me to add error handling.

Reply:
see line 130